radix-engine-interface/src/types/blueprint.rs

```diff
@@ -11,26 +11,24 @@ use scrypto_schema::{InstanceSchema, KeyValueStoreSchema};
 use utils::ContextualDisplay;
 
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
-pub enum ObjectBlueprintInfo {
-    Inner { outer_object: GlobalAddress },
-    Outer,
+pub enum OuterObjectInfo {
+    Some { outer_object: GlobalAddress },
+    None,
 }
 
-impl Default for ObjectBlueprintInfo {
+impl Default for OuterObjectInfo {
     fn default() -> Self {
-        ObjectBlueprintInfo::Outer
+        OuterObjectInfo::None
     }
 }
 
+/// Core object state, persisted in `TypeInfoSubstate`.
 #[derive(Debug, Clone, PartialEq, Eq, ScryptoSbor)]
 pub struct ObjectInfo {
     pub global: bool,
-
     pub blueprint_id: BlueprintId,
-    pub version: BlueprintVersion,
-
-    // Blueprint arguments
-    pub blueprint_info: ObjectBlueprintInfo,
+    pub blueprint_version: BlueprintVersion,
+    pub outer_object: OuterObjectInfo,
```

Comment:
Can you revert these renames? Some of my upcoming PRs refactor `ObjectInfo` a bit.

Reply:
Reverted `blueprint_version`, but kept `outer_object` as I feel it's more accurate.